tspng/test_implantation.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+import pytest
+
+from io import BytesIO
+from tspng.implantation import implant,implant_from_file
+from tspng.extraction import extract
+
+def test_implant():
+    '''
+    Tests the dictionary keys from an example TSPNG file path.
+    '''
+    #test file
+    implant('tests/assets/coco_data.json','tests/assets/empty.png')
+    test_data = extract('tests/assets/empty.png_MOD')
+    assert list(test_data.keys())==['info','licenses','images','annotations','models','categories']
+
+def test_implant_fails():
+    with pytest.raises(TypeError):
+        implant('Test for failure')
```

Comment:
The invocation of the `implant` function is missing the second required argument, the path to the image file.